frontend/src/main/web/src/containers/App.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react'
+// import a11y from 'react-a11y'
+import { connect } from 'react-redux'
+import Helmet from 'react-helmet'
+import {
+  Nav,
+  View,
+  Icons
+} from '../components'
+
+// if (process.env.NODE_ENV === 'development') a11y(React)
```

Comment:
Is this here as a reminder, or just forgotten? If it is a reminder, say `TODO` there so it is clear.